simulator/src/main/java/peersim/kademlia/das/CustomDistributionDas.java

```diff
@@ -211,6 +211,8 @@ public boolean execute() {
 
       } else {
         dasProt = ((DASProtocol) (Network.get(i).getProtocol(protocolDasNonValidatorID)));
+        nonValidatorsIds.add(kadProt.getKademliaNode().getId());
+        node.setServer(false);
```

Comment:
Do we assume that all the non-validators are non-servers?

Reply:
good question. i think we shouldn't set all non-validators are non-servers, othewise won't provide samples. However for the dht implentation we set that (thats why i implemented this option). However if we use this for DHT we should do the same for our das approach, otherwise is not a fair comparison